src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+@import './var';
+@import './main';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 400px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    display: flex;
+    justify-content: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top-img {
+    width: 100%;
+  }
+
+  &__bottom {
+    margin-top: 40%;
+  }
+
+  &__tittle {
```

Comment:
There's a typo in the class name 'card__tittle'. It should be 'card__title' to match the HTML class name.